example-01/vector_addition.cpp

```diff
@@ -78,7 +78,7 @@ int main(int argc, char *argv[]) {
   myQueue.submit(cg);
 
   {
-    const auto write_t = sycl::access::mode::read;
```

Comment:
I think the `accessor` here should still be `access::mode::read` so the change should instead be to change the name `write_t` -> `read_t`.

Reply:
Should be correct now.